Examples/Swift/ViewController.swift

```diff
@@ -346,6 +346,10 @@ extension ViewController: NavigationViewControllerDelegate {
     func navigationViewController(_ navigationViewController: NavigationViewController, didArriveAt waypoint: Waypoint) -> Bool {
         // When the user arrives, present a view controller that prompts the user to continue to their next destination
         // This type of screen could show information about a destination, pickup/dropoff confirmation, instructions upon arrival, etc.
+        
+        //If we're not in a "Multiple Stops" demo, show the normal EORVC
+        guard waypoints.count > 2 else { return true }
```

Comment:
Ideally, this would be based off of route progress and would return false for every waypoint which is not the last.